### PR TITLE
Final Update for v0.10

### PR DESCRIPTION
**Pull request by MacSplody**
- Merge updates from v0.13 to v0.10

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1964,14 +2280,17 @@ plugins:
...
+      - 'Alternate Start - Live Another Life.esp'
       - 'Immersive Citizens - AI Overhaul.esp'
+      - 'Immersive Citizens - OCS patch.esp'
+      - 'Open Cities Skyrim.esp'
   - name: 'ELFX - Hardcore.esp'
     url:
       - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/2424/'
         name: 'Enhanced Lights and FX on Nexus Mods'
-    after:
-      - 'Immersive Citizens - AI Overhaul.esp'
```
Comment (MacSplody) [inline masterlist.yaml:1974]: I had to remove that rule due to a bug in the starting room of Alt start. There is a wild edit in the
 `ELFX - Hardcore.esp` plug-in which alters the bounds of that room. The v0.13 does not yet have that change, and I didn't want to make another commit to v0.10.

**Review by Ortham — APPROVED** [review]
Looks good to me.

**Comment by MacSplody** [comment]
image

```
  5  5     EnhancedLightsandFX.esp
  6  6     Alternate Start - Live Another Life.esp
  7  7     Immersive Citizens - AI Overhaul.esp
  8  8     ELFX - Hardcore.esp
  9  9     Immersive Citizens - ELFXHardcore patch.esp
```
The load order required for the bug to be replicated.
